libV2/config/schema.js

```diff
@@ -115,7 +115,17 @@ const schema = Joi.object({
             return filterObj;
         }, {},
     )),
+    metrics: {
+        enabled: Joi.boolean(),
+        host: Joi.string(),
+        serverPort: Joi.number().port(),
```

Comment:
why do we need to use different metrics port?

a single metrics endpoint can be used to serve the metrics from all these endpoints; and it will serve only the metrics which are actually registered (e.g. if one instance runs only some tasks, it will return only that tasks' metrics)

having a single port to scrape will be slightly better in terms of (prometheus) load ; and simplify configuration/integration (esp. in Kubernetes, where we would need to declare extra port mappings to auto-detection)

Reply:
also, for consistency : the names used here (in config) should somehow match the names of the metrics (which are currently the name of the classes)

Reply:
Forgot to comment yesterday on this, the different ports are needed because the tasks don't run in the same process.